src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -1,23 +1,24 @@
 package edu.oregonstate.cs361.battleship;
 
 import java.util.ArrayList;
+import java.util.Random;
 
 /**
  * Created by michaelhilton on 1/4/17.
  */
 public class BattleshipModel {
 
-    private Ship aircraftCarrier = new Ship("AircraftCarrier",5, new Coordinate(0,0),new Coordinate(0,0));
-    private StealthShip battleship = new StealthShip("Battleship",4, new Coordinate(0,0),new Coordinate(0,0));
-    private StealthShip submarine = new StealthShip("Submarine",3, new Coordinate(0,0),new Coordinate(0,0));
+    private MilitaryShip aircraftCarrier = new MilitaryShip("AircraftCarrier", 5, new Coordinate(0, 0), new Coordinate(0, 0), false);
```

Comment:
I'm a bit lost on what the change from Ship/StealthShip to MilitaryShip will allow us to do. From what I can understand the TA wants it set so that they are grouped under the same parent but then call a child process?

I know you mentioned you're not sure on how to proceed so if you don't know yet that's cool -- hopefully they'll go over this sprint in detail.